src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Utils/SPDXVersionParser.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Utils;
+
+using System;
+
+public static class SPDXVersionParser
+{
+    // SPDX versions are of the form "SPDX-m.n". We only care about the major version.
+    public static bool VersionMatchesRequiredVersion(string spdxVersionString, int requiredMajorVersion)
```

Comment:
This seems like a good candidate for some unit tests

Reply:
Unit tests are in Microsoft.Sbom.Api.Tests.